services/src/main/java/org/keycloak/theme/beans/LocaleBean.java

```diff
@@ -37,12 +37,12 @@ public class LocaleBean {
     private final String currentLanguageTag;
     private final boolean rtl; // right-to-left language
     private final List<Locale> supported;
-    private static final ConcurrentHashMap<java.util.Locale, Boolean> bidiMap = new ConcurrentHashMap<>();
+    private static final ConcurrentHashMap<String, Boolean> bidiMap = new ConcurrentHashMap<>();
```

Comment:
Any particular reason this is no longer a `Locale`?

Reply:
I found that you can already specify in a theme the local name of the locale as a string. By default, it is using Java's internal name for the language, but you can override it. Assuming that you override it for those languages that are not listed in Java or you customize it for any reason, this now uses that String and not the locale.